pandas/io/formats/html.py

```diff
@@ -79,7 +79,7 @@ def _write_cell(self, s, kind='td', indent=0, tags=None):
         self.write(u'{start}{rs}</{kind}>'
                    .format(start=start_tag, rs=rs, kind=kind), indent)
 
-    def write_tr(self, line, indent=0, indent_delta=4, header=False,
+    def write_tr(self, line, indent=0, indent_delta=0, header=False,
```

Comment:
Changing the default argument is an API change.  Could you explain why you are doing this?

Reply:
I think that the only public method of `HTMLFormatter` should be `write_result()`.

The other methods are currently a mix of private and public. There does not seem to be any consistency.

If `write_tr` should be private, the API change would not be an issue. The `indent_delta` is not left to the default by any calls in the current implementation. `indent_delta=0` would be a cleaner default.

Following on from the discussion regarding making `indent` a stateful entity rather than leaving it as a parameter to individual methods: The `write_tr` method accesses the `bold_rows` instance variable. The value should probably be passed through the method's `nindex_levels` parameter which was probably the intention of adding that parameter to the method.

The other HTML formatting methods `write`, `write_th`, `write_td` and `_write_cell` do not access instance variables with the exception that `write` needs access to `elements` which effectively the output buffer.

Reply:
yeah this is ok, as its internal. I agree its less that clear what is external / internal (and we don't usually use _ prefixed). If you want to indicate in the doc-string of an external API would be ok.